src/dotnet/UI.Blazor/Components/Form/Toggle.razor

```diff
@@ -27,6 +33,19 @@
 
     public bool CurrentValue { get; set; }
 
+    protected override void OnAfterRender(bool firstRender) {
+        if (firstRender) {
```

Comment:
I suppose it's to late to make vibration.
I would expect vibration feedback just after I clicked a toggle (or panel with toggle).
Here it's much later.
Why did not implement it in OnParametersSet?